src/config.js

```diff
@@ -1,6 +1,11 @@
 import { config as bskConfig, network as bskNetwork } from 'blockstack'
 import winston from 'winston'
 import fs from 'fs'
+import os from 'os'
+import process from 'process'
+
+const BLOCKSTACK_TEST = process.env.BLOCKSTACK_TEST;
+const DB_PATH = BLOCKSTACK_TEST ? '~/transaction_broadcaster.testnet.db' : '~/transaction_broadcaster.db';
```

Comment:
This is fine by me -- though it's slightly preferable to just write the config to a file -- also I think the semicolons will make the linter angry

Reply:
Fixed!